src/network/serverpackethandler.cpp

```diff
@@ -470,6 +470,11 @@ void Server::process_PlayerPos(RemotePlayer *player, PlayerSAO *playersao,
 
 	v3f position((f32)ps.X / 100.0f, (f32)ps.Y / 100.0f, (f32)ps.Z / 100.0f);
 	v3f speed((f32)ss.X / 100.0f, (f32)ss.Y / 100.0f, (f32)ss.Z / 100.0f);
+	const v3s16 old_node_position = floatToInt(playersao->getBasePosition(), BS);
+	const v3s16 new_node_position = floatToInt(position, BS);
+	if(old_node_position != new_node_position){
+		m_script->on_moveplayer(playersao, new_node_position, old_node_position);
+	}
```

Comment:
Since the action of the conditional is 1 line you don't need curly brackets.

Reply:
Missing space after `if`

Reply:
Okay, will fix